gamemodes/terrortown/gamemode/client/cl_weapon_pickup.lua

```diff
@@ -45,13 +11,3 @@ net.Receive("ttt2_switch_weapon_update_cache", function()
         WSWITCH:UpdateWeaponCache()
     end)
 end)
-
--- register a binding for the weapon switch, the default should be the use key
-bind.Register(
-    "ttt2_weaponswitch",
-    AttemptWeaponSwitch,
-    nil,
-    "header_bindings_ttt2",
-    "label_bind_weaponswitch",
-    input.GetKeyCode(input.LookupBinding("+use") or KEY_E)
-)
```

Comment:
targetid.lua is still using this

Reply:
good catch, targetID would have to be updated as well then. Also the language file needs updating!

Reply:
![grafik](https://github.com/TTT-2/TTT2/assets/72046466/2a0b8c26-7d3b-48b2-a3e8-b8ca0eab8000)
Is it?

Reply:
Oh yeah I can see where...

Reply:
yes, looking at entities always used "+use", but look at a weapon which uses the pickup binding

Reply:
> Oh yeah I can see where... ![grafik](https://private-user-images.githubusercontent.com/72046466/309555955-2295d54d-79c6-4bec-b582-8e1bcc65b85f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk0NzU1MTAsIm5iZiI6MTcwOTQ3NTIxMCwicGF0aCI6Ii83MjA0NjQ2Ni8zMDk1NTU5NTUtMjI5NWQ1NGQtNzljNi00YmVjLWI1ODItOGUxYmNjNjViODVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzAzVDE0MTMzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVhMWRmZTYzNjJlMjE3NDQzMWFhNWM3YjU1OGFmODQzY2FiMDUzYmYxM2I1MWE3OWQyN2E1MTJkMWQ4NjllYTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uD2chJdyFHkn79KXobYn0NjGCxbIB4dkSlD2gmfmiWA)

well **THIS** is a bug you just accidentally fixed with your rework. These two bindings should be the same :D

Reply:
![grafik](https://github.com/TTT-2/TTT2/assets/72046466/ca9a4454-8d4f-430f-9723-4235018f8a77)
Yeah just noticed that as well

Reply:
should be a a easy change though as all external addons already use "+use", only our builtin weapon pickup uses something different